src/models/chart-model/index.js

```diff
@@ -13,6 +13,7 @@ export default function createChartModel({
   model,
 }) {
   let interactionInProgess = false;
```

Comment:
I think we can make this a little cleaner. I suggest:
Get rid of the "panEnd" variable, I think it will be enough to only have "interactionInProgress". And that variable is more suitable to have in the view-handler I think. So, remove panEnd, and move interactionInProgress into view-handler.